Assets/Scripts/API/Save/CharacterRecord.cs

```diff
@@ -262,6 +265,7 @@ public struct CharacterRecordData
             public DaggerfallSkills skills;
             public Int16[] skillUses;
             public UInt32[] equippedItems;
+            public Races race2; // Stores character's original race for when returning from being a vampire, werewolf or wereboar
```

Comment:
@Allofich I'm hoping you might be able to help with some trouble I'm having with importing lycanthrope saves. When importing a classic character infected as either a werewolf/wereboar, I've noticed that race/race2 will both be the same value (e.g. Breton) unless our lycanthrope is actively transformed at the time save was made. If the character is not transformed at time of save, I can't seem to find a way to identify if they are a werewolf/wereboar.

Is this something you might know more about, so I can check weretype on import even if not transformed at time of save? Thank you. :)

Reply:
I commented in https://github.com/Interkarma/daggerfall-unity/commit/dd4980dd828bbaabc872199f066bd4c955aaec07